code/game/machinery/bots/cleanbot.dm

```diff
@@ -29,23 +29,21 @@
 	var/next_dest
 	var/next_dest_loc
 
-/obj/machinery/bot/cleanbot/New()
```

Comment:
atomise

Reply:
Eh it's not that bad of an idea to do this, personally I hate making 3 PRs for one simple, coherent change